lib/Sema/CSDiagnostics.cpp

```diff
@@ -348,6 +348,16 @@ bool RequirementFailure::isStaticOrInstanceMember(const ValueDecl *decl) {
   return decl->isStatic();
 }
 
+bool WrappedValueMismatch::diagnoseAsError() {
+  auto *locator = getLocator();
+  auto elt = locator->castLastElementTo<LocatorPathElt::WrappedValue>();
+
+  emitDiagnostic(diag::composed_property_wrapper_mismatch,
+                 resolveType(getFromType()), elt.getType(), getToType());
```

Comment:
Nit: `getFromType()` would call `resolve` internally so there is no need to call it again. But I think we need to call that for `elt.getType()`?

Reply:
I agree. I did some investigating, and it turns out it's not required to use resolveType for any of the three function calls, so I went on and got rid of it all together, I hope that was okay.

Reply:
I think it is required on the element because element would get a type that might have type variables in it, so we'd end up printing `_` instead of an actual type type variable was bound to.

Reply:
Oh that makes sense, I will fix that!